modules/form-field/react/lib/Label.tsx

```diff
@@ -16,19 +17,32 @@ const labelStyles = [
     padding: 0,
   },
   (props: LabelProps) => {
```

Comment:
Nit: I know it's not something you touched here, but do you mind destructuring the props like we do in other components? (i.e. `({labelPosition, required}) => ({`)